src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2945,8 +2945,11 @@ class S4Importer final : public IParkImporter
     {
         const auto originalString = _s4.string_table[(stringId - USER_STRING_START) % 1024];
         std::string_view originalStringView(originalString, USER_STRING_MAX_LENGTH);
-        auto withoutFormatCodes = RCT12::RemoveFormatCodes(originalStringView);
-        return rct2_to_utf8(withoutFormatCodes, RCT2_LANGUAGE_ID_ENGLISH_UK);
+        auto asUtf8 = rct2_to_utf8(originalStringView, RCT2_LANGUAGE_ID_ENGLISH_UK);
+        auto buf = std::make_unique<char[]>(asUtf8.size() + 1);
+        std::strcpy(buf.get(), asUtf8.data());
```

Comment:
Why can't this be string with pointer to utf8 data and it's length? You still need to convert it to string upon returning